packages/story-editor/src/components/publishModal/mainContent/index.js

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import {
+  Button,
+  BUTTON_SIZES,
+  BUTTON_TYPES,
+  Text,
+} from '@googleforcreators/design-system';
+import styled from 'styled-components';
+import { __ } from '@googleforcreators/i18n';
+/**
+ * Internal dependencies
+ */
+import MandatoryStoryInfo from './mandatoryStoryInfo';
+import StoryPreview from './storyPreview';
+
+const Main = styled.div`
+  display: grid;
+  min-width: 917px;
+  width: 100%;
+  height: 100%;
+  margin: 0 -3.7% 0 3.7%;
+  grid-gap: 3.7%;
+  /* grid-template-columns: 230px 276px 308px; */
+  grid-template-columns: 25% 30% 34%;
+  grid-template-rows: auto;
+  grid-template-areas:
+    'preview mandatory panel'
+    'footer footer panel';
+`;
```

Comment:
💭  can be updated in the future, but is there a way to update the grid areas to be contained in the modal? I don't think there should be a need to overflow the content horizontally. Vertically yes we'll need to scroll.

![image](https://user-images.githubusercontent.com/22185279/152058198-6adbe2c5-173c-48d0-a8b3-8a5a4565cc1d.png)

Reply:
🤔 Wondering if it makes more sense to put the margin on `StoryPreview` so you don't have to fiddle with negative margins?

Reply:
yeah! I anticipate this gets tweaked for overflow once the panels are in here. I didn't really want to fiddle with it too much ahead of that because it seemed redundant. 

Re the margin on `StoryPreview`, it gets a little weird because of the `grid` which wants margins this way and we need to have it work the same in RTL. Ideally there's no negative margin - so same answer as above, kinda just waiting for the final pieces to land.